week-2/implement/vat.js

```diff
@@ -8,3 +8,10 @@
 // Given a number,
 // When I call this function with a number
 // it returns the new price with VAT added on
+
+function vat(num) {
+  const numWithoutMoneySymbols = num.substring(1, num.length);
+  return numWithoutMoneySymbols * 1.2;
+}
+console.log("Your new price with VAT is", vat("£40"));
```

Comment:
You've called your function with a string data type - but you only need to consider number inputs.